src/SprykerEco/Zed/UnzerGui/UnzerGuiConfig.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+/**
+ * MIT License
+ * For full license information, please view the LICENSE file that was distributed with this source code.
+ */
+
+namespace SprykerEco\Zed\UnzerGui;
+
+use Spryker\Zed\Kernel\AbstractBundleConfig;
+
+class UnzerGuiConfig extends AbstractBundleConfig
```

Comment:
[minor] Protected constants should go after the public.

Reply:
no public constants